css/index.css

```diff
@@ -84,8 +82,6 @@ nav {
     width: 75%;
 }
 
-
-
 .banner {
     width: 880px;
     margin-bottom: 3%;
```

Comment:
so you  would end up only having index.css and everything would load this style sheet